code/datums/components/clown_disbelief_item.dm

```diff
@@ -1,4 +1,5 @@
 var/global/list/image/clown_disbelief_images
+var/global/list/image/clown_disbelief_clown_mobs //Subscribed list of mobs that are currently deemed to be clowns
```

Comment:
I'd doc comment this, and make it slightly more explicit what the list is recording.
I.e. `/// An associative list of clown disbelief item subscription counts, indexed by the mob they've been applied to.`
To me, this makes it clear that if you pass a mob to the list as an index, you're getting back a subscription count.